python/ribasim/ribasim/models.py

```diff
@@ -218,27 +220,27 @@ class UserTime(BaseModel):
 
 
 class Root(BaseModel):
-    BasinProfile: BasinProfile | None
-    BasinState: BasinState | None
-    BasinStatic: BasinStatic | None
-    BasinTime: BasinTime | None
-    DiscreteControlCondition: DiscreteControlCondition | None
-    DiscreteControlLogic: DiscreteControlLogic | None
-    Edge: Edge | None
-    FlowBoundaryStatic: FlowBoundaryStatic | None
-    FlowBoundaryTime: FlowBoundaryTime | None
-    FractionalFlowStatic: FractionalFlowStatic | None
-    LevelBoundaryStatic: LevelBoundaryStatic | None
-    LevelBoundaryTime: LevelBoundaryTime | None
-    LinearResistanceStatic: LinearResistanceStatic | None
-    ManningResistanceStatic: ManningResistanceStatic | None
-    Node: Node | None
-    OutletStatic: OutletStatic | None
-    PidControlStatic: PidControlStatic | None
-    PidControlTime: PidControlTime | None
-    PumpStatic: PumpStatic | None
-    TabulatedRatingCurveStatic: TabulatedRatingCurveStatic | None
-    TabulatedRatingCurveTime: TabulatedRatingCurveTime | None
-    TerminalStatic: TerminalStatic | None
-    UserStatic: UserStatic | None
-    UserTime: UserTime | None
+    basinprofile: BasinProfile | None = None
```

Comment:
Shouldn't this be `basin_profile`?

Reply:
It's unused, the whole Root is there just to generate all child models from this single root definition.json.